Pearson-Advance/frontend-app-institution-portal
# refactor: instructor avatar and how it is displayed

# Description
In this PR is updated the way in how the instructors are displayed.

## Ticket
https://agile-jira.pearson.com/browse/PADV-1242


## Change log
- Removed default instructor image.
- Removed .keep file in assets ( not longer required ).
- Fixed typos in test description.
- New component to handle the avatar of the instructors.


## Screenshots
### Before
![be](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/00778c0c-d8ca-4bc6-bda1-be4c44b9398a)


### After
![af](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/f1c4565b-4b4c-4c4b-9523-4a8fc1953187)

## How to test

Clone the Institution Portal MFE
```Bash
     git clone <url>
```
Install the packages 📦.
```Bash
     npm i
```
Start project.
```Bash
     npm start
```

Then go to "classes" and pick any class to see the details of it.
